src/Makefile.test.include

```diff
@@ -205,7 +206,7 @@ test/data/%.raw.h: test/data/%.raw
 	@$(MKDIR_P) $(@D)
 	@{ \
 	 echo "namespace raw_tests{" && \
-	 echo "static unsigned const char $(*F)[] = {" && \
+   echo "static unsigned const char $(*F)[] = {" && \
```

Comment:
Pls revert this one too https://github.com/UdjinM6/dash/commit/7d2eb46d97bf4ffe88dc97324d5f27de51c2ddaf (my bad 🙈 )

Reply:
Resolved in 4b2b5f7